pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -118,17 +128,34 @@ pub mod pallet {
 		/// The sender account that will be used in the OutboundQueue
 		/// implementation.
 		#[pallet::constant]
-		type Sender: Get<Self::AccountId>;
+		type Sender: Get<DomainAddress>;
 
 		/// Type used for queueing messages.
-		type MessageQueue: MessageQueue<Message = GatewayMessage<Self::AccountId, Self::Message>>;
+		type MessageQueue: MessageQueue<Message = GatewayMessage<Self::Message, Self::RouterId>>;
+
+		/// Maximum number of routers allowed for a domain.
+		#[pallet::constant]
+		type MaxRouterCount: Get<u32>;
+
+		/// Type for identifying sessions of inbound routers.
+		type SessionId: Parameter
+			+ Member
+			+ BaseArithmetic
+			+ Default
+			+ Copy
+			+ MaybeSerializeDeserialize
+			+ TypeInfo
```

Comment:
Super NIT. I think we do not need `MaybeSerializeDeserialize` here and `TypeInfo` is under Parameter IIRC

Reply:
Will try removing it, thank you!